app/components/primer/alpha/overlay.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= show_trigger_button %>
+<div class="Overlay-backdrop <%= @backdrop_classes %>">
```

Comment:
Generally speaking Ruby style is to use 2 spaces for indentation and indent blocks inside `if` statements, etc.